home-manager/home-manager.nix

```diff
@@ -1,78 +1,64 @@
-{ pkgs ? import <nixpkgs> {}
-, confPath
-, confAttr ? null
-, check ? true
-, newsReadIdsFile ? null
-}:
+{ pkgs ? import <nixpkgs> { }, confPath, confAttr ? null, check ? true
+, newsReadIdsFile ? null }:
```

Comment:
I preferred the oldversion, is that tweakable in nixfmt ?

Reply:
No :/ https://github.com/serokell/nixfmt/issues/46
We could keep the exception for this file.

Reply:
There are some improvements in the formatting but the degradations annoy me more than the improvements so personnally I would advocate to keep the same nixfmt version, just remove the exceptions. Let's see what others think